doc/api/cli.md

```diff
@@ -218,6 +218,9 @@ Support is currently only provided for the root context and no guarantees are
 currently provided that `global.Array` is indeed the default intrinsic
 reference. Code may break under this flag.
 
+`--require` runs prior to freezing intrinsics in order to allow polyfills to
```

Comment:
are polyfills usually added with `--require`? Have you considered freezing after the first-tick? Or even just exposing a `freeze()` js API, so the timing is up to the user?

Reply:
First tick usually includes most of the app, including things you may not wish to allow mutating the intrinsics. The point is to contain the possible mutations, leaving it up to a programmatic API would prevent a clear way of enforcing a policy of what is allowed to mutate the intrinsics, so it also wouldn't be a good choice.